en/docs/develop/restapis/association.yaml

```diff
@@ -148,49 +153,293 @@ paths:
         500:
           description: Server Error
           schema:
-             $ref: '#/definitions/Error'
+            $ref: '#/definitions/Error'
       tags:
-      - me
-  /me/associations/switch:
-    put:
+        - me
+
+  /me/associations/{associated-user-id}:
+    delete:
       parameters:
-        -
-          name : SwitchUserReqeust
-          in : body
+        - name: associated-user-id
+          in: path
           required: true
+          type: string
+      summary: Delete given user account from my user associations
+      description: |
+        This API is used to delete the given user account from the associations of the auhtentiated user.
```

Comment:
authenticated

Reply:
Addressed in 861bcaec1512c2d43091b40273eaf2c2dce11f3e